src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.System/Properties/Resources.resx

```diff
@@ -120,6 +120,9 @@
   <data name="confirm_system_commands" xml:space="preserve">
     <value>Show a dialog to confirm system commands</value>
   </data>
+  <data name="Localize_system_commands" xml:space="preserve">
+    <value>Localize system commands</value>
```

Comment:
```suggestion
    <value>Localized system commands instead of english ones</value>
```
Might be better because it syas that...
- otherwise English is used.
- on English local the setting has "no function".